recipes/markdown-it-py/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "markdown-it-py" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "925da55cba665cb9808ef851885993f856c678c8009077a59ab3134d7dac876f"
+
+build:
+  number: 0
+  entry_points:
+    - markdown-it = markdown_it.cli.parse:main
+    - markdown-it-bench = markdown_it.cli.benchmark:main
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
For future reference, only `script: "{{ PYTHON }} -m pip install . -vv"` is enough here..